front/app/components/IdeasMap/IdeaMapCard.tsx

```diff
@@ -232,20 +235,14 @@ const IdeaMapCard = memo<Props>(
           tabIndex={0}
         >
           {smallerThanMaxTablet && (
-            <CloseButtonWrapper>
-              <CloseButton
-                width="26px"
-                height="26px"
-                padding="0px"
-                buttonStyle="secondary"
-                icon="close"
-                iconSize="12px"
-                onClick={handleCloseButtonClick}
-              />
-              <ScreenReaderOnly>
-                <FormattedMessage {...messages.a11y_hideIdeaCard} />
-              </ScreenReaderOnly>
-            </CloseButtonWrapper>
+            <StyledCloseIconButton
+              widthInPx={12}
+              heightInPx={12}
+              onClick={handleCloseButtonClick}
+              a11y_buttonActionMessage={messages.a11y_hideIdeaCard}
```

Comment:
In fact the map overview of ideas is not accessible, so this message doesn't matter a lot. If it did, I would have to use getInputTermMessage to get the correct copy (based on what's specified for the project/phase).